.storybook/stories/playground/dashboard/index.js

```diff
@@ -31,10 +30,6 @@ export default {
   title: 'Playground/Dashboard',
 };
 
-const AppContainer = styled.div`
-  height: 100vh;
```

Comment:
To clarify, we don't need a `max-height: 100vh` for the dashboard?

Reply:
I tried, by adding a lot of stories in the dashboard without `max-height` in storybook, it didn't have any issue.